.env.qa

```diff
@@ -31,3 +30,7 @@ NEXT_PUBLIC_RNS_REGISTRY_ADDRESS=0x7d284aaac6e925aad802a53c0c69efe3764597b8
 # CR-related env variables
 NEXT_PUBLIC_BACKERS_MANAGER_ADDRESS=0x41841e316F85933247fC7b51c79B76F22eB239bc
 NEXT_PUBLIC_REWARD_DISTRIBUTOR_ADDRESS=0x83Eac3Abe6AAF4a4AAe8B067888FAD600c4cF6B3
+
+# Rif Wallet Services
+NEXT_PUBLIC_RIF_WALLET_SERVICES=https://dev.rws.app.rootstockcollective.xyz
+NEXT_PUBLIC_EVENTS_FROM_BLOCK=5784028
```

Comment:
Where does this number come from?
Neither the [backers manager](https://explorer.testnet.rootstock.io/address/0x41841e316f85933247fc7b51c79b76f22eb239bc) or the [reward distributor](https://explorer.testnet.rootstock.io/address/0x83eac3abe6aaf4a4aae8b067888fad600c4cf6b3) were deployed at this block 🤔